savu/plugins/reconstructions/fista_recon.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2014 Diamond Light Source Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+.. module:: fista_recon
+   :platform: Unix
+   :synopsis: Wrapper around FISTA iterative algorithm in FISTA-tomo
+
+.. moduleauthor:: Daniil Kazantsev <scientificsoftware@diamond.ac.uk>
+"""
+
+from savu.plugins.reconstructions.base_recon import BaseRecon
+from savu.data.plugin_list import CitationInformation
+from savu.plugins.driver.gpu_plugin import GpuPlugin
+
+import numpy as np
+# install FISTA-tomo with: conda install -c dkazanc fista-tomo
```

Comment:
Probably a todo for the future, but we should move this to a DLS or CCPi repo just for longevity